src/Analyzers/CSharp/Analyzers/MakeLocalFunctionStatic/MakeLocalFunctionStaticHelper.cs

```diff
@@ -26,11 +26,22 @@ public static bool TryGetCaputuredSymbols(LocalFunctionStatementSyntax localFunc
                 return false;
             }
 
+            // If other local functions area called the it can't be made static
+            // unless they themselves are static, or the local function is recursive
+            if (dataFlow.UsedLocalFunctions.Any(lf => !lf.IsStatic && lf.DeclaringSyntaxReferences.FirstOrDefault()?.GetSyntax() != localFunction))
```

Comment:
That seems unrelated to actually getting captured symbols, no? Just because it calls a local function doesn't mean it doesn't have any captured symbols.

Reply:
You are correct, but thats why I made it private, so nothing outside the context of the various "Make Local Function Static" functions can call it. TryGetCapturedSymbolsIfCanBeMadeStatic seemed like an odd name to me, but happy to rename it if people think its confusing, or have better names.